src/test/java/LIC0Test.java

```diff
@@ -0,0 +1,111 @@
+import org.junit.jupiter.api.Test;
```

Comment:
Wrong indentation in this file. It should use two-space indenation, this one uses more

Reply:
Fixed!